### DISCUSSION TITLE
[feature] When scrolling with the scroll indicator, the photos should be kept aligned to the month

### DISCUSSION THREAD

**Discussion by teaalltr** [immich-app/immich#9680] [Feature Request]
### The bug

So basically in Google Photos when you scroll up and down using the scroll indicator, the photos are kept aligned with the name of the current month being scrolled always in the same place on the screen, about in the middle of the view.
This aids searching because the brain has the time to process the month being scrolled cause the eye doesn't need to move.
I think this absolutely need to be implemented in Immich too, it's very useful for usability and accessibility possibly

### The OS that Immich Server is running on

Nn

### Version of Immich Server

Nn

### Version of Immich Mobile App

1.105.1

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
nn
```

### Your .env content

```Shell
nn
```

### Reproduction steps

```bash
1.nn
2.
3.
...
```

**Comment by alextran1502** [comment]
For UI requests, can you include a screenshot? Thanks! I will convert this to a Feature Request

**Reply by teaalltr** [reply]
@alextran1502 Sure, here's a comparison between Immich and Google Photos. Note the month name at about one third of the screen that keeps staying there in Google Photos, while in Immich it doesn't.

https://github.com/immich-app/immich/assets/11365978/2e07b310-8e59-407e-bbd8-852e6362088c


https://github.com/immich-app/immich/assets/11365978/8b0fdfd5-6240-4f16-9024-37852610f555
